proto/options.proto

```diff
@@ -482,4 +482,7 @@ message Options {
     uint32 matrix_locations = 54;                                  // a one to many or many to one time distance matrix. Does not affect
   }                                                                // sources_to_targets when either sources or targets has more than 1 location
                                                                    // or when CostMatrix is the selected matrix mode.
+  oneof has_elevation_interval {
```

Comment:
do we need to differentiate between default and not set? if not we can remove the `oneof`

Reply:
Does this default to 0.0 in the case where elevation_interval is not included in the JSON request? If so, then I think we can remove oneof.

Reply:
yep it does https://protobuf.dev/programming-guides/proto3/#default

Reply:
pushed a fix. I kept code to make sure the value is in range when pbf inputs are used.